src/main/java/ch/uzh/ifi/hase/soprafs22/controller/UserController.java

```diff
@@ -60,6 +61,10 @@ public ResponseEntity<UserGetDTO> createUser(@RequestBody UserPostDTO userPostDT
     MultiValueMap<String, String> headers = new HttpHeaders();
     headers.set("token", createdUser.getToken());
 
+    System.out.println("created: " + createdUser.getName());
+    System.out.println(createdUser.getToken());
+    System.out.println(createdUser.getId());
+
```

Comment:
Didn't SonarLint complain already about that one?

Reply:
yes, you are right, i overlooked them